.github/workflows/test-sui.yaml

```diff
@@ -139,7 +139,55 @@ jobs:
 
       ###### Command: Multisig ######
 
-      # TODO: Add multisig tests
+      - name: Init Multisig
+        run: |
+          # Create new addresses
+          sui client new-address secp256k1 multisig1
+          sui client new-address secp256k1 multisig2
+
+          # Export keys and addresses
+          KEY_1=$(sui keytool export --key-identity multisig1 --json | jq -r .key.publicBase64Key)
+          KEY_2=$(sui keytool export --key-identity multisig2 --json | jq -r .key.publicBase64Key)
+          ADDRESS_1=$(sui keytool export --key-identity multisig1 --json | jq -r .key.suiAddress)
+          ADDRESS_2=$(sui keytool export --key-identity multisig2 --json | jq -r .key.suiAddress)
+
+          # Get multisig address
+          MULTISIG_ADDRESS=$(sui keytool multi-sig-address --pks $KEY_1 $KEY_2 --weights 1 1 --threshold 1 --json | jq -r .multisigAddress)
+
+          # Initialize multisig
+          node sui/multisig.js --action init --threshold 1 --base64PublicKeys $KEY_1 $KEY_2 --schemeTypes secp256k1 secp256k1
+
+          # Faucet operations
+          node sui/faucet.js --recipient $ADDRESS_1
+          node sui/faucet.js --recipient $ADDRESS_2
```

Comment:
Isn't the multisig address the one that needs funding and not the participants?

Reply:
nice catch! thanks